examples/src/main/python/ml/dataframe_example.py

```diff
@@ -17,7 +17,7 @@
 
 """
 An example of how to use DataFrame for ML. Run with::
-    bin/spark-submit examples/src/main/python/ml/dataframe_example.py <input>
+    bin/spark-submit examples/src/main/python/ml/dataframe_example.py <dataset>
```

Comment:
Shall we change `dataset` to like .. `input_path` or `path`? I think the name `dataset` is usually used for `Dataset` instance.